README.md

```diff
@@ -1,2 +1,95 @@
 # moveit_example_apps
-Demonstrator applications for using MoveIt with full robotic setups
+
+This project demonstrates advanced applications for using MoveIt with full robotic setups. For each listed application, there is a tutorial introducing how to replicate and run the application either in simulation or with real robot. If you are interested in the technique details, you can visit [MoveIt Tutorials](https://ros-planning.github.io/moveit_tutorials/) for further information.
+
+## Applications
+
+| Application Name | Documentation          | Docker |
+| ---------------- | -----------------------| ------ |
+| Random Pick      | [random_pick_tutorial] | WIP    |
+
+[random_pick_tutorial]: https://roboticsyy.github.io/moveit_example_apps/doc/random_pick/random_pick.html
+
+## Continuous Integration
+
+Work in progress.
+
+## Contributing
+
+There are many diverse application examples of what you can use MoveIt for. Your contribution is encouraged, no matter the application is developed for some usages, competitions or research topics, or to demonstrate the newly developed MoveIt feature.
+
+If you are interested in adding a new MoveIt application in this project, please consider following the below instructions.
+
+### Directory Structure
+
+For each example application, it should contain two parts: example codes and tutorials.
+
+* The example codes of each application should live in its own subdirectory in the `./examples/<app_name>` direcotry.
```

Comment:
```suggestion
* The example codes of each application should live in its own subdirectory in the `./examples/<app_name>` directory.
```

Reply:
I will rectify all the `directory` in all the documents.